src/core/reference.cc

```diff
@@ -7,9 +7,7 @@ PythonReference::PythonReference(PythonObject *scope) : mScope(scope) {
 }
 
 PythonReference::~PythonReference() {
```

Comment:
[F] This `Unref` was causing the same errors seen shown in https://github.com/nodejs/node-addon-api/issues/998. Removing the `Unref` gets hide of the errors. The explanation is shown in the issue [here](https://github.com/nodejs/node-addon-api/issues/998#issuecomment-846593458)